packages/flame_tiled/lib/src/flame_tsx_provider.dart

```diff
@@ -5,7 +5,8 @@ import 'package:xml/xml.dart';
 
 /// A implementation of [TsxProvider] use by RenderableTileMap.
 ///
-/// It uses [Flame.bundle] or a custom asset bundle and has a built-in cache for the file read.
+/// It uses [Flame.bundle] or a custom asset bundle
+/// and has a built-in cache for the file read.
```

Comment:
This is fine, but ideally it should only break once it reaches 80 and not before.

Reply:
Hmm, okay. Should I change that?
For me it makes sense to break if a new part of the sentence has started

Reply:
It's a bit like using a line break as a comma, something one wouldn't do in a book for example.
Anyhow this is fine, but now you know until the next PR. :)